Product/Production/Deploy/ear/weblogic/pom.xml

```diff
@@ -19,11 +19,6 @@
         </dependency>
         <dependency>
             <groupId>org.apache.geronimo.specs</groupId>
-            <artifactId>geronimo-jaxws_2.2_spec</artifactId>
```

Comment:
Ok.  so you're removing this in addition to reverting the mail jars?  Just confusing since this wasn't  part of the description.